src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountSteps.java

```diff
@@ -35,4 +39,33 @@ public void iFillTheFieldsAndPressTheSaveButton(final Map<AccountInputs, String>
         values.keySet().forEach(step -> modifyPage.getStrategyStepMap(values).get(step).fillField());
         modifyPage.getSaveNewAccountButton();
     }
+
+    /**
+     * @param endPoint is the endpoint.
+     * @param values   are the map values.
+     */
+    @Given("^I post a new Account to \"([^\"]*)\" endpoint$")
+    public void iPostANewAccountToEndpoint(final String endPoint, final Map<String, String> values) {
+//        APIRequestManager.post(values, APICommons.buildEndPoint(endPoint));
+        APIRequestManager.post(values, endPoint);
+    }
+
+    /**
+     * Method to save response.
+     *
+     * @param variableName the key to save.
+     */
+    @And("^save the response as \"([^\"]*)\"$")
+    public void saveTheResponseAs(final String variableName) {
+        APICommons.saveResponse(variableName, APIRequestManager.getResponse());
+    }
+
+    /**
+     * @param endPoint is the endpoint.
+     */
+    @When("^I delete an Account with \"([^\"]*)\" endpoint$")
+    public void iDeleteAnWithEndpoint(final String endPoint) {
+        //APIRequestManager.delete(APICommons.buildEndPoint(endPoint));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)